dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/program/EnrollmentStoreTest.java

```diff
@@ -244,16 +247,16 @@ void testGetWithScheduledNotifications() {
     // Queries
     List<Enrollment> results;
     // A
-    results = enrollmentStore.getWithScheduledNotifications(a1, today);
+    results = programNotificationService.getEnrollmentsWithScheduledNotifications(a1, today);
```

Comment:
not sure what is tested here but should this not move as well?

Reply:
yes, this should be moved and some point and the method be made private

Reply:
I think `testGetWithScheduledNotifications` could be moved to `ProgramNotificationServiceTest`.